client/src/components/categorical/category/index.js

```diff
@@ -22,6 +22,8 @@ import maybeTruncateString from "../../../util/maybeTruncateString";
   };
 })
 class Category extends React.Component {
+  _labelWidth = globals.leftSidebarWidth - 80;
```

Comment:
Why is this an instance variable, and not just a variable in the outer (file) closure?